__tests__/app.test.js

```diff
@@ -177,3 +178,71 @@ describe("/api/articles", () => {
     });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+    test("GET: 200 - responds with an array of comments for the given article_id", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments).toBeInstanceOf(Array);
+                if(comments.length === 0){
+                    return;
+                }
+                comments.forEach(comment => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
+                    });
+                });
+            });
+    });
+
+    test("GET: 400 - responds with an error for an invalid article_id", () => {
+        return request(app)
+            .get("/api/articles/invalid_id/comments")
+            .expect(400)
+            .then(({ body }) => {
+                expect(body).toEqual({ message: "Invalid ID" });
+            });
+    });
+    test("GET: 200 - responds with an empty array if no comments exist for an article_id", () => {
+        return request(app)
+            .get("/api/articles/2/comments") 
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments).toEqual([]); 
+            });
+    });
+
+    test("GET: 200 - comments are sorted by most recent first", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments).toBeInstanceOf(Array);
+                comments.forEach(comment => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
+                    });
+                });
```

Comment:
You've already tested these assertions above so you don't need it again here. Since your test is just about how the comments are sorted, you only need to test for that.